README.md

```diff
@@ -1 +1,158 @@
+</div>
+
+# 📗 Table of Contents
+
+- [� Table of Contents](#-table-of-contents)
+- [📖 \[Portfolio\] ](#-portfolio-)
+  - [🛠 Built With ](#-built-with-)
+    - [Tech Stack ](#tech-stack-)
+    - [Key Features ](#key-features-)
+  - [🚀 Live Demo ](#-live-demo-)
+  - [💻 Getting Started ](#-getting-started-)
+    - [Prerequisites](#prerequisites)
+    - [Setup](#setup)
+    - [Install](#install)
+    - [Usage](#usage)
+  - [👥 Authors ](#-authors-)
+  - [🔭 Future Features ](#-future-features-)
+  - [🤝 Contributing ](#-contributing-)
+  - [⭐️ Show your support ](#️-show-your-support-)
+  - [🙏 Acknowledgments ](#-acknowledgments-)
+  - [📝 License ](#-license-)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 [Portfolio] <a name="about-project"></a>
+
+**[Leaderboard - APIS]** in this project we are using APIS to take a list of users and scores
+
+## 🛠 Built With <a name="built-with"></a>
+
+- HTML
+- CSS
+- JAVASCRIPT
+
+### Tech Stack <a name="tech-stack"></a>
+
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="https://html.spec.whatwg.org/">HTML</a></li>
+    <li><a href="https://www.w3.org/Style/CSS/Overview.en.html">CSS</a></li>
+  </ul>
+</details>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+- **[In this project we have make a brand new modular system using ES6 syntax and a boundler like Webkit]**
+- **[You could find a dinamic website]**
+- **[CSS styles]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- [Live Demo Link]()
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to clone this project you need:
+
+◦ Installation of Git
+◦ Installation of NPM
+◦ A working browser
+◦ Internet connection
+◦ A code editor like Visual Studio Code
+◦ Installation of Live Server plugging in Visual Studio Code
+
+### Setup
+
+To setup and install this project, follow the below steps:
+
+- Clone this project by the command:
+
+```
+$ git clone https://github.com/SaveryIV/Awesome-Books-Project.git
+```
+
+- Then switch to the project folder by the bellow query:
+
+```
+$ cd dir
+```
+
+### Install
+
+For setting up the linters you need:
+
+-Follow the steps mentionated in this article https://github.com/microverseinc/linters-config/tree/master/html-css
+
+### Usage
+
+To run the project:
+
+-Download the extension Live Server for Ritwick Dey.
+-Then close and open again VScode.
+-When you open the folder of the project in your local machine, in the lower left of your screen, you gonna see a item what says "Go Live".
+-Press that button and set the developers tools and select your toggle device toolbar.
+-And enjoy the project!.
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 👥 Authors <a name="authors"></a>
+
+👤 **Agustin Soria Meza**
+
```

Comment:
- [ ]  [OPTINAL] It could be better to add links to your social media, to facilitate the user to get more information about you 👍